route/build/src/write_restart_pio.f90

```diff
@@ -871,6 +948,41 @@ SUBROUTINE write_IRFbas_state(ierr, message1)
 
   END SUBROUTINE write_IRFbas_state
 
+  SUBROUTINE write_bas_tracer_state(ierr, message1)
```

Comment:
Description.

Reply:
subroutine name is changed to `write_bas_solute_state`.  The same question as reading state, wondering how much descriptions are needed here. This and a bunch of other same routines in state writing module are writing states (in restarts file). Do you think more detailed descriptions are needed here and the other routines?

Reply:
I added short, general description for `write_state_nc`